packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/instances.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
```

Comment:
what about creating a file common.dart where we can place widgets that are used in different places of the memory screen? Not every class needs its own file like other languages

Reply:
'Not every class needs its own file' does not mean 'put all small classes in one huge library'.
For me it means 'Put classes into one library if they serve shared goal and thus can be seen as one unit of functionality, and file is of reasonable size and this is observable'.

Example of such library with many classes is instance_set_button.dart